app/Policies/CommentPolicy.php

```diff
@@ -26,6 +28,37 @@ public function view(?User $user, Comment $comment): bool
         return true;
     }
 
+    public function create(User $user, ?Model $commentable): bool
```

Comment:
There appear to be separate policies for each of the supported model types. Not sure how those should (or do) interact with this.
* AchievementCommentPolicy
* GameCommentPolicy
* NewsCommentPolicy
* UserCommentPolicy

Maybe this is just for the other types: like ticket, hashes, and claims.

Reply:
That was my assumption too - `CommentPolicy` is just a catch-all for "everything else". The other policies shouldn't have any bearing on this one.